doc/api/fs.md

```diff
@@ -5478,7 +5465,7 @@ the file contents.
 [`fs.mkdir()`]: #fs_fs_mkdir_path_options_callback
 [`fs.mkdtemp()`]: #fs_fs_mkdtemp_prefix_options_callback
 [`fs.open()`]: #fs_fs_open_path_flags_mode_callback
-[`fs.opendir()`]: #fs_fs_opendir_path_options_callback
+[`fs.opendir()`]: #fs_fs_opendir_path__callback
```

Comment:
Should there only be one underscore after `path`?

Reply:
Weird, I wonder why local lint didn't catch that

Reply:
https://travis-ci.com/nodejs/node/jobs/243949295#L305-L321 wasn't the `lint` target that failed, it was the `doc-only` target that builds the HTML version of the docs. On the CI the failure occurs as part of the build job (not the linter job), e.g. https://ci.nodejs.org/job/node-test-commit-linuxone/nodes=rhel72-s390x/16184/.